airbyte-config/persistence/src/main/java/io/airbyte/config/persistence/DatabaseConfigPersistence.java

```diff
@@ -1575,17 +1583,47 @@ public void replaceAllConfigs(final Map<AirbyteConfig, Stream<?>> configs, final
       result.put(ConfigSchema.STANDARD_SOURCE_DEFINITION.name(),
           standardSourceDefinitionWithMetadata
               .stream()
-              .map(ConfigWithMetadata::getConfig)
-              .map(Jsons::jsonNode));
+              .map(configWitMetadata -> {
```

Comment:
just to confirm, this doesn't mess up auto migration, right? i don't remember all the places `dumpConfigs` gets called.

Reply:
I am not sure about what is the auto migration. From the usage I found with the find usage of IntelliJ, it looked ok. I will add reviewer to double check that.

Reply:
did a little hunting and i agree, it should be safe.